src/api-client.ts

```diff
@@ -20,87 +14,63 @@ const enum ContentTypes {
     SRT = 'application/x-subrip'
 }
 
+// tslint:disable-next-line
+const sdkVersion = require('../package.json').version;
+
 export class RevAiApiClient {
     accessToken: string;
-    version: string;
-    instance: AxiosInstance;
+    apiHandler: ApiRequestHandler;
     constructor (accessToken: string, version = 'v1') {
+        this.apiHandler = new ApiRequestHandler(`https://api.rev.ai/revspeech/${version}/`, {
+            'Authorization': `Bearer ${accessToken}`,
+            'User-Agent': `RevAi-NodeSDK/${sdkVersion}`
+        });
         this.accessToken = accessToken;
-        axios.defaults.baseURL = `https://api.rev.ai/revspeech/${version}/`;
-        /* tslint:disable:no-string-literal */
-        axios.defaults.headers.common['Authorization'] = `Bearer ${accessToken}`;
-        axios.defaults.headers['User-Agent'] = `RevAi-NodeSDK/2.0.1`;
-        /* tslint:enable:no-string-literal */
     }
 
     async getAccount(): Promise<RevAiAccount> {
-        try {
-            const response = await axios.get('/account');
-            return response.data;
-        } catch (error) {
-            switch (error.response.status) {
-                case 401:
-                    throw new RevAiApiError(error);
-                default:
-                    throw error;
-            }
-        }
+        return await this.apiHandler.makeApiRequest<RevAiAccount>(
+            'get',
```

Comment:
We don't need to put each parameter on a new line.

You can put them all on the same line and wrap only the parameters that overflow the line character limit